### PR TITLE
add downstream test report

### PR DESCRIPTION
**Pull request by bryevdv**
fixes: #7720 

For now at least this test does not fail, it only generates a report. Want to avoid a situation where one or other project is intentionally out out of sync (major revision) and then the test can't be made to pass at all in any way for some interval

### CODE REVIEW

**Comment by bryevdv** [comment]
@philippjfr @jlstevens are there any tests you would suggest adding?

cc @mrocklin

**Comment by philippjfr** [comment]
Apologies, I meant to reply to this ages ago. I think running our bokeh plotting unit tests would be sensible, I don't think you'd ever want to run our notebook tests due to the complexity and run times involved. Not sure what the best way for you to do so is though since we don't ship the unit tests inside the holoviews package.

**Comment by mrocklin** [comment]
> Not sure what the best way for you to do so is though since we don't ship
the unit tests inside the holoviews package.

Are the tests very large?  If not then you might choose to change this
policy.

**Comment by philippjfr** [comment]
>Are the tests very large?  If not then you might choose to change this policy.

They aren't and I've argued this point a few times. I'll restart that discussion.

**Comment by bryevdv** [comment]
Ok well we should merge this now then, and circle back after HV has a release with tests

**Comment by bryevdv** [comment]
@mattpap do you have any comments?

File: scripts/ci/test:downstream [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e # exit on error
+set -x # echo commands
+
+pushd `python -c "import site; print(site.getsitepackages()[0])"`
+py.test distributed/bokeh
```
Comment (mattpap) [inline scripts/ci/test:downstream]: Given `-e` flag, this will fail this job when test fail (or am I missing something?).

Comment (bryevdv) [reply scripts/ci/test:downstream]: Ah, yes apparently so. I thought it meant something slightly different.

**Pull request merged**